rpcs3/Emu/Cell/PPUInterpreter.cpp

```diff
@@ -150,12 +150,13 @@ const g_ppu_scale_table;
 
 bool ppu_interpreter::MFVSCR(PPUThread& ppu, ppu_opcode_t op)
 {
-	throw std::runtime_error("MFVSCR" HERE);
+	LOG_WARNING(PPU, "MFVSCR is unimplemented");
```

Comment:
I think an exception would make much more sense, as it's quite a serious error/problem.